src/index.js

```diff
@@ -1,6 +1,8 @@
 import './style.css';
 import { getLocalstorage } from './modules/localstorage.js';
 import EnterEvent from './modules/EnterEvent.js';
+import deleteTask from './modules/delete.js';
 
 getLocalstorage();
 EnterEvent();
+deleteTask();
```

Comment:
* As pointed out by the previous reviewer it is still impossible to delete a particular task (ie a delete icon or button can be added to the dom for each task created so that when clicked it removes that particular task and the object array in local storage is updated ) [refer back to the project requirement ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20deleting%20a%20task%20(remove%20an%20element%20from%20the%20array).)
* Also i see the `deleteTask` function performs the feature of removing all completed tasks ( tasks with checkbox marked ✅ ) and not deleting a particular task these are two separate required functionality and should have different functions for that [Refer here to confirm these requirements ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Create%20a%20new,use%20filter()%20method).)